hangman/lib/hangman/application.ex

```diff
@@ -0,0 +1,13 @@
+defmodule Hangman.Application do
+    use Application
+
+    def start(_type, _args) do
+        import Supervisor.Spec, warn: false
+        children = [ 
```

Comment:
# -10

So this is wonky. You're supervising the API module, not the game server.

The better approach it to have the application supervise the Hangman.Server. It creates a new game in its init function. The API calls for make_move and tally then call this server.